src/lib/request.js

```diff
@@ -1,17 +1,41 @@
-import $ from 'jquery';
-
+import 'whatwg-fetch';
+import { stringify } from './query-string';
 /**
  * Checks whether or not the current method passed in is valid
  *
- * @param {string} method
- * @returns {boolean}
+ * @param {String} method
+ * @returns {Boolean}
  */
-function isValidHTTPMethod(method) {
-    const allowedMethods = ['GET', 'POST', 'PUT', 'DELETE'];
+const isValidHTTPMethod = (method) => ['GET', 'POST', 'PUT', 'DELETE'].indexOf(method) !== -1;
 
-    return allowedMethods.indexOf(method) !== -1;
-}
+/**
+ * Checks if requested template is using sections
+ *
+ * @param {Object|Array|String} requestedTemplate
+ */
+const isUsingSections = (requestedTemplate) => typeof(requestedTemplate) === 'object' && !Array.isArray(requestedTemplate);
 
+/**
+ * Returns templates array
+ *
+ * @param {Object|Array|String} requestedTemplate
+ * @returns {Array}
+ */
+const getTemplates = (requestedTemplate) => {
+    let templates = [];
+    if (isUsingSections(requestedTemplate)) {
+        for (const template in requestedTemplate) {
```

Comment:
I guess we can replace this `for..in` with just `templates = Object.values(requestedTemplate)`

Reply:
We could, but we would need to shim this for IE. I don't want for 1 line add a polyfill.
Also there are couple places, where we can improve, but first we need to update dev dependencies.

Reply:
@jairo-bc Ok, I thought since we are using babel it should transpile everything, but if it's outdated and requires extra efforts let's skip these non-significant changes